worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Controller.scala

```diff
@@ -485,6 +485,7 @@ private[deploy] class Controller(
               s"${committedReplicaIds.size()} committed replica partitions, " +
               s"${emptyFileReplicaIds.size()} empty replica partitions, " +
               s"${failedReplicaIds.size()} failed replica partitions.")
+          storageManager.persistShuffle(shuffleKey, true)
```

Comment:
IMO in order not to block normal execution, we should add the persist task in a queue and let a dedicated thread do the persist job.

Reply:
Is this method thread safe?

Reply:
https://github.com/google/leveldb/blob/main/doc/index.md#concurrency

According to level DB own docs, it's thread-safe in one process if not using writeBatch and I didn't use it.